src/lib/operators/table_scan/column_between_table_scan_impl.cpp

```diff
@@ -49,7 +49,6 @@ void ColumnBetweenTableScanImpl::_scan_non_reference_segment(
     for (const auto& sorted_by : chunk_sorted_by) {
       if (sorted_by.column == _column_id) {
         _scan_sorted_segment(segment, chunk_id, matches, position_filter, sorted_by.sort_mode);
-        ++num_chunks_with_binary_search;
```

Comment:
With this PR, this metric will be different because it is not incremented for every `_scan_sorted_segment` but only for the ones without early out. Is anybody relying on this metric (@Bouncner maybe?).

Reply:
Fine with me.